src/Nixfmt/Pretty.hs

```diff
@@ -103,7 +103,7 @@ prettyTerm (List (Ann paropen Nothing []) [ListItem item] parclose)
         = pretty paropen <> pretty item <> pretty parclose
 
 prettyTerm (List (Ann paropen trailing leading) items parclose)
-    = pretty paropen <> pretty trailing <> line
+    = nest 0 $ pretty paropen <> pretty trailing <> line
```

Comment:
What is the purpose of `nest 0`?

Reply:
I'll add a comment to the definition of `nest` to explain this.